DataStructures/SortedLinkedList/SortedLinkedList.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DataStructures.SortedLinkedList
+{
+    /// <summary>
+    /// TODO.
+    /// </summary>
+    /// <typeparam name="T">TODO. 2.</typeparam>
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6332606)